todoman/cli.py

```diff
@@ -171,10 +171,12 @@ def cli(ctx, color, porcelain):
 @click.argument('summary', nargs=-1)
 @click.option('--list', '-l', callback=_validate_list_param,
               help='The list to create the task in.')
+@click.option('--location',
+              help='The location of the todo.')
```

Comment:
Please add this to `_todo_property_options`, such that this is also automatically added to `todo edit`.